python/cudf/setup.py

```diff
@@ -46,12 +50,19 @@
             "../../cpp/build/include",
             "../../thirdparty/cub",
             "../../thirdparty/libcudacxx/include",
+            ARROW_HOME + "/include",
+            DLPACK_INCLUDE,
             os.path.dirname(sysconfig.get_path("include")),
             np.get_include(),
             cuda_include_dir,
         ],
-        library_dirs=[get_python_lib(), os.path.join(os.sys.prefix, "lib")],
-        libraries=["cudf"],
+        library_dirs=[
+            get_python_lib(),
+            os.path.join(os.sys.prefix, "lib"),
+            ARROW_HOME + "/lib",
+            cuda_lib_dir,
+        ],
+        libraries=["cudf", "cudart", "cuda", "nvrtc"],
```

Comment:
Do we need to add Arrow as a library here as well? I imagine Arrow Cython that needs the headers may need the shared library as well.

Reply:
That would make sense. In fact that could potentially solve having to build the python libs in Arrow in " -DARROW_PYTHON=ON" I'll give that a shot. Keep in mind builds on a Xavier take awhile but will report back when it finishes

Reply:
These aren't Python libs, these are C++ shared libraries needed for building the Python extensions defined here.

As mentioned above we'll soon have GPU conda packages where we'll remove our vendored Cython and just rely on those packages, so we shouldn't need to worry about handling Arrow  in here as long as the user installed the packages properly.

Reply:
Makes sense, no arrow here then.